src/pd/Client.cc

```diff
@@ -75,9 +75,11 @@ Client::~Client()
 
 void Client::update(const std::vector<std::string> & addrs, const ClusterConfig & config_)
 {
-    uninit();
```

Comment:
I think we can remove `uninit` method?

Reply:
Do you mean remove the `uninit` method and move the code in previous `uninit` method into dtor?

Reply:
Yes